coralproject/talk
# passes scrape_max_response_size to scaper's FetchOptions

## What does this PR do?
Fixes a bug where the `scrape_max_response_size` environment variable is not used.

## What changes to the GraphQL/Database Schema does this PR introduce?
None.

## How do I test this PR?
1. To reproduce this bug, direct Coral to scrape a page that's greater than 2mb and smaller than scrape_max_response_size (which defaults to 10mb) & verify that you see an error in the app logs like this:

```js
    err: {
      "message": "SCRAPE_FAILED: content size at https://slate.com/comments/news-and-politics/2021/09/texas-ban-abortion-fund-lawsuit.html over limit: 2000000",
      "name": "ScrapeFailed",
```

I did this by loading the embed in my browser and setting the storyURL to https://slate.com/comments/news-and-politics/2021/09/texas-ban-abortion-fund-lawsuit.html

2. On this branch, verify that you can scrape the above page.
